tendermint/vue
# fix: Capitalize the itemName

## Description & Context

All requests to the blog item use camel case naming conventions and are strictly dependent on them. For now, we allow names that can start with a lowercase letter and break requests. We now convert the `itemName` property to start with uppercase.

## How To Test

1. Open the [Data.view](https://github.com/tendermint/vue/blob/develop/packages/template/src/views/Data.vue) template;
2. For `item-name` prop try using the name like `item-name="modelName"` or `item-name="ModelName"`;
3. In both cases the module should work;

## Linked issue

Closes #201
